tools/templates/sdl.yml

```diff
@@ -10,13 +10,13 @@ jobs:
     - msbuild
     - npm
   steps:
-  - task: UseDotNet@2
-    displayName: 'Install .NET Core SDK'
-    inputs:
-      packageType: sdk
-      version: 2.2.x
-      includePreviewVersions: false
-      installationPath: $(Agent.ToolsDirectory)/dotnet
+  # - task: UseDotNet@2
+  #   displayName: 'Install .NET Core SDK'
+  #   inputs:
+  #     packageType: sdk
+  #     version: 3.0.x
+  #     includePreviewVersions: false
```

Comment:
forgotten?

Reply:
We intent on bringing back explicit installation of .Net Core 3.0 once all projects are migrated to .Net Core 3.0. Added a comment about that.